linode/lke/resource.go

```diff
@@ -440,3 +445,13 @@ func customDiffValidateOptionalCount(ctx context.Context, diff *schema.ResourceD
 
 	return nil
 }
+
+func getExternalPoolTags(d *schema.ResourceData) []string {
+	var externalPoolTags []string
+	if v, ok := d.GetOk("external_pool_tags"); ok {
+		for _, tag := range v.(*schema.Set).List() {
+			externalPoolTags = append(externalPoolTags, tag.(string))
+		}
+	}
+	return externalPoolTags
+}
```

Comment:
You can use the `helper.ExpandStringSet(...)` helper to simplify this:
https://github.com/linode/terraform-provider-linode/blob/dev/linode/helper/expand.go#L15-L17

Reply:
done